packages/wallets/jestTransformer.js

```diff
@@ -0,0 +1,28 @@
+const babelJest = require('babel-jest').default;
+
+module.exports = {
+    process(sourceText, sourcePath, options) {
+        const transformer = babelJest.createTransformer({
+            babelrc: false,
+            configFile: false,
+            plugins: [
+                ['@babel/plugin-proposal-decorators', { legacy: true }],
+                ['@babel/plugin-proposal-class-properties', { loose: true }],
+                ['@babel/plugin-proposal-private-methods', { loose: true }],
+                ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
+                '@babel/plugin-proposal-export-default-from',
+                '@babel/plugin-proposal-object-rest-spread',
+                '@babel/plugin-proposal-export-namespace-from',
+                '@babel/plugin-syntax-dynamic-import',
+                ['@babel/plugin-proposal-unicode-property-regex', { useUnicodeFlag: false }],
+                '@babel/plugin-proposal-optional-chaining',
+                '@babel/plugin-proposal-nullish-coalescing-operator',
+            ],
```

Comment:
Configured jest config separately from babel config to exclude the newly added `babel-plugin-transform-barrels` plugin which causes the imports to be unrecognizable in unit and integration tests

<img width="486" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/437d5aed-897e-4119-ad4e-4eb5465db2bd">